src/components/pages/program-view/ProgramViewContainer.jsx

```diff
@@ -15,34 +18,47 @@ const Container = styled.div`
 `;
 
 const ProgramViewContainer = props => {
-  //Props passed from Program Dashboard
-  const { programData } = props;
+  //Program Id passed from Main Dash param in url
+  const { id } = useParams();
+
+  //Redux State Managers
+  const dispatch = useDispatch();
+
+  const { programName, programDescription, programType, courses } = useSelector(
+    state => state.programs.program
+  );
+
+  //Dispatch Action to Load Program Info
+  useEffect(() => {
+    dispatch(programsActions.getProgramThunk(id));
+  }, []);
 
   return (
     <>
       {/*Course Title Section*/}
       <TitleContainer>
         <div>
-          <h2>{programData.programName}</h2>
-          <h3>{programData.programType}</h3>
+          <h2>{programName}</h2>
+          <h3>{programType}</h3>
         </div>
         <div>
           <Button buttonText="Edit" />
         </div>
       </TitleContainer>
 
       {/*Course Description Section*/}
-      <Container>{programData.programDescription}</Container>
+      <Container>{programDescription}</Container>
 
       {/*Render Course Components Section*/}
       <Container>
         <h4>Courses</h4>
-        {programData.courses.map(data => (
+
+        {courses.map(data => (
```

Comment:
let's use a more clear variable than `data`. `course` or `courseData` would work

Reply:
Totally! I agree I should be better about being more descriptive with my name convention